reference.md

```diff
@@ -801,6 +822,41 @@ more details about the macro interface.
 (print (my-max 20 20))
 ```
 
+### `macro` define a single macro
+
+```fennel
+(macro my-max [x y]
+  `(let [x# ,x y# ,y]
+     (if (< x# y#) y# x#)))
+```
+
+If you are only defining a single macro, this is equivalent to the
+previous example. The syntax mimics `fn`.
+
+### `macrodebug` print the expansion of a macro
+
+```fennel
+(macrodebug (-> abc
+                (+ 99)
+                (> 0)
+                (when (os.exit))))
+; -> (if (> (+ abc 99) 0) (do (os.exit)))
+```
+
+Call the `macrodebug` macro with a form and it will expand all the
+macros in that form and print out the resulting form. Note that the
```

Comment:
This seems to imply that all macros in the form will be expanded, but this isn't quite accurate, since what really happens is that it repeatedly expands each macro form until it the top-level form is no longer a macro form, then prints it.

Even if we don't expand it fully, maybe we can come up with a terse way to hint that it won't expand nested subforms? Just so people don't expect a fully expanded/"atomic" form.

Reply:
That's a great point; I hadn't even considered that crucial distinction. I've changed it to make it clearer that it only affects top-level calls.